libr/include/r_flag.h

```diff
@@ -46,6 +46,7 @@ typedef struct r_flag_item_t {
 	char *color;    /* item color */
 	char *comment;  /* item comment */
 	char *alias;    /* used to define a flag based on a math expression (e.g. foo + 3) */
+	bool section_end; /* is a section_end flag */
```

Comment:
umf, nope, flags are something generic, shouldnt depend on section_end. as i said i think its better to kill those _end flags and use the section size field